packages/blade/src/components/Menu/_decisions/decisions.md

```diff
@@ -0,0 +1,308 @@
+# Menu
+
+Action Menu displays a list of actions on temporary surfaces. They allow users to action(s) from multiple options. They appear when users interact with a button, action, or other control.
+
+Unlike Dropdown, they are specifically made to be non-selectable, clickable items and are more flexible with what they can contain. Checkout [Menu vs Dropdown](#menu-vs-dropdown) for more context.
+
+<img alt="Menu Overview Image" src="overview.png" width="300px" />
+
+## Design
+
+- [Figma - Menu](https://www.figma.com/design/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=90082-41948&m=dev)
+
+## API
+
+```jsx
+<Menu>
+  {/* Any interactive item which has onClick, onMouseDown exposed */}
+  <Button>Open Menu</Button>
```

Comment:
Let's define what needs to be exposed for a functional Menu Trigger. 

- Events (onClick, onMouseOver)
- Accessibility props (aria-haspopup, aria-controls)
- ref for floating positioning etc etc

Reply:
Added it to doc